package.json

```diff
@@ -27,8 +28,7 @@
         "archiver": "0.4.10"
     },
     "scripts": {
-        "postinstall": "grunt build",
-        "test": "grunt test"
+        "build": "cd src && npx shifter --walk"
```

Comment:
Should we also navigate back a directory after running shifter?

Reply:
Not really, because the commands here don't affect the shell from where you are running "npm run build". It would be necessary if we executed more commands, only.